### DISCUSSION TITLE
Discussion topics for Mar 7, 2022 Engineering Sync

### DISCUSSION THREAD

**Discussion by carlosfelix2** [CDCgov/prime-reportstream#4618] [Show and tell]
1. Initial discussion on how to make changes to the API without impacting users
2. Updated dependabot playbook

**Comment by whytheplatypus** [comment]
How to represent mapping between two specifications (kind of: declarative vs. procedural)

Reference:
- [declarative example](https://github.com/CDCgov/prime-reportstream/pull/4594/files#diff-122ee67f6c74219b93d8f9800c6b4520bc031da425d2bdc00ce8045b5a8eb360)
- [procedural example](https://github.com/CDCgov/prime-reportstream/pull/4594/files#diff-29d416a4b73e3cd84290e3abb4ceb48e87c8a8236b163f003b64be4c32a4eaaa)

**Comment by carlosfelix2** [comment]
Topics related to changing the API without impacting users:
1. Detect what clients are using the API, so we can contact them
2. Deprecate the old API
3. Test old and new APIs
4. Contacting the users when there are breaking changes
